src/modules/SignIn/components/SignIn/components/FormSignIn/FormSignIn.jsx

```diff
@@ -46,35 +49,80 @@ export default class FormSignIn extends Component {
     onSubmit(event) {
         event.preventDefault();
         const { formData } = this.state;
-        const validateResult = validSignInForm(formData);
-
-        if (validateResult.status) {
-            API.signIn(formData)
-                .then((response) => {
-                    if (response.status === signInSuccessCode) {
-                        setTokens(response.data.access, response.data.refresh);
-                        window.location.href = "/dashboard";
-                    }
-                })
-                .catch((error) => {
-                    if (error.response.status === signInErrorCode) {
-                        this.setState(() => ({
-                            errorMsg: signInErrorMsg,
-                        }));
+
+        API.signIn(formData)
+            .then((response) => {
+                if (response.status === 200) {
+                    setTokens(response.data.access, response.data.refresh);
+                    window.location.href = "/dashboard";
+                }
+            })
+            .catch((error) => {
+                if (error.response.status === 400) {
+                    let incorrectStatus = false;
+
+                    if (Object.keys(error.response.data).includes("non_field_errors")) {
```

Comment:
You can replace a conditional expression with `const incorrectStatus = Object.keys(error.response.data).includes("non_field_errors");`.